stetho/src/main/java/com/facebook/stetho/inspector/protocol/module/CSS.java

```diff
@@ -96,30 +96,7 @@ public JsonRpcResult getMatchedStylesForNode(JsonRpcPeer peer, JSONObject params
         params,
         GetMatchedStylesForNodeRequest.class);
 
-    final GetMatchedStylesForNodeResult result = new GetMatchedStylesForNodeResult();
-
-    final RuleMatch match = new RuleMatch();
-    final RuleMatch accessibilityMatch = getAccessibilityRuleMatch();
-    result.matchedCSSRules = ListUtil.newImmutableList(match, accessibilityMatch);
-
-    match.matchingSelectors = ListUtil.newImmutableList(0);
-
-    Selector selector = new Selector();
-    selector.value = "<this_element>";
-
-    CSSRule rule = new CSSRule();
-
-    rule.origin = Origin.REGULAR;
-    rule.selectorList = new SelectorList();
-
-    rule.selectorList.selectors = ListUtil.newImmutableList(selector);
-
-    rule.style = new CSSStyle();
-    rule.style.cssProperties = new ArrayList<>();
-
-    match.rule = rule;
-
-    rule.style.shorthandEntries = Collections.emptyList();
+    final Map<String, List<CSSProperty>> selectors = new HashMap<>();
```

Comment:
and, we may as well type the `Map`'s `V`alue as `ArrayList` instead of `List` -- doing this can sometimes enable certain static optimizations